libc/config/baremetal/api.td

```diff
@@ -85,5 +85,10 @@ def TimeAPI : PublicAPI<"time.h"> {
 }
 
 def UCharAPI : PublicAPI<"uchar.h"> {
-  let Types = ["mbstate_t"];
+  let Types = [
+    "mbstate_t",
```

Comment:
you should also add these types to the `api.td` for `linux` if you're adding this header to the list of linux headers.

Reply:
What about windows, I couldn't find any header list. Does this mean, all are included by default, or none of them?